apps/nextjs-app-router/src/app/[lang]/[[...path]]/layout.tsx

```diff
@@ -13,10 +14,15 @@ export default function RootLayout({
   return (
     <html lang={params.lang}>
       <head>
-        <DraftModeScript />
+        <DraftModeScript appOrigin={process.env.MAKESWIFT_APP_ORIGIN} />
```

Comment:
Add `appOrigin` for debugging w/ local builder, see https://makeswifthq.slack.com/archives/C014SCS4G75/p1725915902722189